### ISSUE TITLE
The newly updated list still isn’t working

### ISSUE THREAD

**Issue by githubbuserrr** [CapedCrusader42/IfunnyAdBlocker#11]
Thank you for all the work you’ve put into helping strangers, but even the newly updated list does not work. The banner and comment ads are gone, but the ads on the main page you see by scrolling are still appearing every once in a while. Thanks for updating, but there’s still something wrong

**Comment by CapedCrusader42** [comment]
Are you using the website or the mobile app?

**Comment by githubbuserrr** [comment]
@CapedCrusader42  i’m using the app version, i’m on ios

**Comment by CapedCrusader42** [comment]
Are you using pihole or lockdown?

**Comment by githubbuserrr** [comment]
@CapedCrusader42 i’m using lockdown

**Comment by CapedCrusader42** [comment]
Do you have a method to view domains that are being sent from your phone?

**Comment by githubbuserrr** [comment]
@CapedCrusader42 i don’t really understand. I don’t believe i have a way of seeing what domains are being sent.  on lockdown i can only see which of the domains that are being visited that i have on the block list, and only the ones on the block list

**Comment by CapedCrusader42** [comment]
Yes, if you can view the domain log, you can use a method of trial and error to block the right domain. It's the same way that I do for my lists. Just ensure that you only access Ifunny and disregard any domains related to other services. Whenever you do find those if you want to add a commit to the list or send them here, I can add them.

**Comment by githubbuserrr** [comment]
@CapedCrusader42 ok, thanks for the help

**Comment by githubbuserrr** [comment]
@CapedCrusader42 i don’t mean to bother you, but the view log on lockdown is only showing me things i have already blocked. Do you know of any apps or programs that would let me see everything that’s being sent from my phone?

**Comment by CapedCrusader42** [comment]
I don't know anything that could be useful for iPhones. Try looking around on Reddit for what people recommend, I would hate to recommend something that I've never used. Alternatively, you can pick up a raspberry pi 0 for almost nothing and go through to install pi-hole. The method itself is super easy and a step-by-step is on their website or there are a ton of youtube videos that walk through it. When you get that all you have to do is copy the link and it adds the domains for you. Then set up a VPN (not sure for iPhone) that connects home and you have a mobile auto-updated adblocker that works for any domains you put in it (general website ads, ifunny, and any other website) compatible with all devices connected on the network. Its through Pi-hole on my own raspberry that I can view the domain queries.

**Comment by githubbuserrr** [comment]
@CapedCrusader42 so it would seem that there is actually a built in domain log in the iphone settings, and with a little fine tuning, i have stopped the ads on ifunny. (At least i hope lol) Thanks for your help man

**Comment by savedude** [comment]
Where do you find the iPhone domain logs?

**Comment by githubbuserrr** [comment]
@savedude go to settings. Go to privacy and scroll all the way down and click on app privacy report.and if you go to app network activity and click on ifunny, there will be a list of domains. From there you can block the domains and be ad free. But do not block the domains that have ifunny in the domain, as they are required for ifunny to work

**Comment by CapedCrusader42** [comment]
Alright, if it has helped your ads please add them to a pull request and I can merge them. Or just let me know and I can do that.
